docs/book/features/file_volume.md

```diff
@@ -15,6 +15,8 @@ In order to utilize this feature in your vSphere environment, you need to make s
 
 - Establish a dedicated file share network connecting all the kubernetes nodes and make sure this network is routable to the vSAN File Share network.  Refer to [Network Access of vSAN File Share](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-EFC00FFF-E720-44F1-B229-4C13687E6B85.html) to understand the setup better.
 
+- Configure the kubernetes nodes with the same DNS server as the one used to configure the file services in the vSAN cluster configuration. This will help the nodes resolve the access points it gets from the File service and mount the file share as a volume while creating a pod.
```

Comment:
Can you explain how?

Reply:
We should also add example commands using  `nsupdate - Dynamic DNS update utility`.

```
# nsupdate
> update add h10-193-8-51.vsanfs3.testdomain 84600 A 10.193.8.51
> send
```

Reply:
Can you also describe how to get these DNS entries? the screenshot will help.

![Screen Shot 2020-09-22 at 4 15 42 PM](https://user-images.githubusercontent.com/22985595/93946838-2bc5fb80-fcef-11ea-8ba3-e7b4f390ecae.png)

Reply:
as per the above example, make sure the node also has a DNS server in the /etc/resolve.conf file

```
# cat /etc/resolv.conf 
# Dynamic resolv.conf(5) file for glibc resolver(3) generated by resolvconf(8)
#     DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN
# 127.0.0.53 is the systemd-resolved stub resolver.
# run "systemd-resolve --status" to see details about the actual nameservers.
nameserver 10.138.127.219 
.
.
```

also show example commands how users can ensure DNS is configured correctly.

```
# ping -c 3 vsanfs3.testdomain
PING vsanfs3.testdomain (10.138.127.219) 56(84) bytes of data.
64 bytes from vsanfs-ad3.vsanfs3.testdomain (10.138.127.219): icmp_seq=1 ttl=121 time=35.2 ms
64 bytes from vsanfs-ad3.vsanfs3.testdomain (10.138.127.219): icmp_seq=2 ttl=121 time=35.1 ms
64 bytes from vsanfs-ad3.vsanfs3.testdomain (10.138.127.219): icmp_seq=3 ttl=121 time=35.1 ms

--- vsanfs3.testdomain ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 2002ms
rtt min/avg/max/mdev = 35.122/35.205/35.297/0.071 ms
```


```
# ping -c 3 h10-193-6-168.vsanfs3.testdomain
PING h10-193-6-168.vsanfs3.testdomain (10.193.6.168) 56(84) bytes of data.
64 bytes from 10.193.6.168 (10.193.6.168): icmp_seq=1 ttl=64 time=0.837 ms
64 bytes from 10.193.6.168 (10.193.6.168): icmp_seq=2 ttl=64 time=0.940 ms
64 bytes from 10.193.6.168 (10.193.6.168): icmp_seq=3 ttl=64 time=0.935 ms

--- h10-193-6-168.vsanfs3.testdomain ping statistics ---
3 packets transmitted, 3 received, 0% packet loss, time 2001ms
rtt min/avg/max/mdev = 0.837/0.904/0.940/0.047 ms
```

Reply:
In addition to this, I also had to override the default DNS policy. without this, I was not able to create the pod, even though networking on the node was correct.

```
      dnsPolicy: "None"
      dnsConfig:
        nameservers:
          - 10.138.127.219
        searches:
          - vsanfs3.testdomain
```